wallet/wallet.go

```diff
@@ -2709,17 +2709,37 @@ func (w *Wallet) ListUnspent(minconf, maxconf int32,
 	return results, err
 }
 
+type ListLeasedOutputResult struct {
+	wtxmgr.LockedOutput
+	Value    int64
+	PkScript []byte
+}
+
 // ListLeasedOutputs returns a list of objects representing the currently locked
 // utxos.
-func (w *Wallet) ListLeasedOutputs() ([]*wtxmgr.LockedOutput, error) {
-	var outputs []*wtxmgr.LockedOutput
+func (w *Wallet) ListLeasedOutputs() ([]*ListLeasedOutputResult, error) {
+	var results []*ListLeasedOutputResult
 	err := walletdb.View(w.db, func(tx walletdb.ReadTx) error {
 		ns := tx.ReadBucket(wtxmgrNamespaceKey)
-		var err error
-		outputs, err = w.TxStore.ListLockedOutputs(ns)
+		outputs, err := w.TxStore.ListLockedOutputs(ns)
+
+		for _, output := range outputs {
+			details, err := w.TxStore.TxDetails(ns, &output.Outpoint.Hash)
+			if err != nil {
+				return err
+			}
+			result := &ListLeasedOutputResult{
+				LockedOutput: *output,
+				Value:        details.MsgTx.TxOut[output.Outpoint.Index].Value,
```

Comment:
nit: line too long. Extract `details.MsgTx.TxOut[output.Outpoint.Index]` as a variable to shorten?